freetextresponse/freetextresponse.py

```diff
@@ -446,18 +474,21 @@ def submit(self, data, suffix=''):
                 )
             )
         self.student_answer = data['student_answer']
-        if self.max_attempts == 0:
-            self.count_attempts = 1
-        else:
-            self.count_attempts += 1
-        self._compute_score()
+        word_count_message = self._get_word_count_message(1)
+        if self._word_count_valid():
```

Comment:
If the count is invalid, then we do not make the student's response count as an attempt, and we do not update the student's score.